helix-term/src/commands.rs

```diff
@@ -186,6 +186,9 @@ impl Command {
         extend_till_prev_char,
         extend_prev_char,
         replace,
+        switch_case,
+        to_uppercase,
+        to_lowercase,
```

Comment:
Should we make it `switch_uppercase` to make it more consistent?

Reply:
Sure.